src/condor_tests/test_ft_plugin_testing.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env pytest
+
+#---------------------------------------------------------------
+# Author: Joe Reuss
+#
+# This test is intended to test file transfer plugins testing mode 
+# using the links file:// and https://. This is to show that with a test
+# on a file that exists, it will still show "file" as a current working plugin
+# and on a url that does not exist it will not show "https" as a current working
+# plugin.
+#
+# Note: this test should not be run on Windows as this feature does not currently
+# work on Windows
+#---------------------------------------------------------------
+
+from ornithology import *
+import os
+import time
+
+
+#
+# Create a test file for the file transfer plugin to use as test
+#
+@standup
+def createTestFile(test_dir):
+    testFile = test_dir / "testFile.txt"
+    testFile_contents = "This is a test"
+    file = open(testFile, "w")
+    file.write(testFile_contents)
+    file.close()
+
+# Set up a personal condor with test_url defined in config:
+@standup
+def condor(test_dir, createTestFile):
+    dir = "log"
+    parent_dir = test_dir / "../../local_dir"
+    path = os.path.join(parent_dir, dir)
+    os.mkdir(path)
+    with Condor(
+        local_dir = test_dir / "condor",
+        config = {
+            "STARTER_DEBUG" : "D_FULLDEBUG",
+            "FILE_TEST_URL" : f"file://{test_dir}/testFile.txt", # give it a file that does exist
```

Comment:
If you add `return testFile` to the `createTestFile` function then you can do:
`"FILE_TEST_URL" : f"file://{createTestFile}"`
This way we only have to worry about changing the test file name once rather than in two different places.
Up to you if you want to change this or not